src/applications/post-911-gib-status/containers/IntroPage.jsx

```diff
@@ -81,18 +63,25 @@ export class IntroPage extends React.Component {
 
     const content = this.getContent();
     return (
-      <div className="row">
-        <div className="medium-8 columns">
-          {this.props.serviceAvailability === SERVICE_AVAILABILITY_STATES.up && gibsWarning}
-          <h1>Post-9/11 GI Bill Statement of Benefits</h1>
-          <p>
-            If you served on active duty after September 10, 2001, you and your dependents may qualify for Post-9/11 GI Bill education benefits. These benefits can help cover all or some of the costs for school or training. If you’ve been awarded Post-9/11 GI Bill benefits, you can review your GI Bill Statement of Benefits and track the amount of time you have left to use for your education or training.
-          </p>
+      <div>
+        <BrandConsolidation>
           {content}
-          <br/>
-          <br/>
-          <br/>
-        </div>
+        </BrandConsolidation>
```

Comment:
The BrandConsolidation elements has to be totally outside the rest of the elements, because the detail-page template contains the page header as part of the content.